src/registrar/migrations/0079_federalagency.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.10 on 2024-03-22 22:18
+
```

Comment:
Blocking: I am assuming this was architected to use a data migration because we'd need to edit this stuff on the fly on stable, correct?

If so, I suggest naming this migration 0079_create_federal_agencies_v01 and documenting how this stuff gets updated in the future in developer docs, similar to how we did documentation for updating user groups.

Reply:
Great call out! Addressed in [this commit](https://github.com/cisagov/manage.get.gov/pull/1928/commits/8b2beba87950e8ef5a13f57effcc3551732efe34)